.vscode/cspell.json

```diff
@@ -175,6 +175,7 @@
     "sdk/remoterendering/azure-mixedreality-remoterendering/**",
     "sdk/schemaregistry/azure-data-schemaregistry/**",
     "sdk/schemaregistry/azure-data-schemaregistry-apacheavro/**",
+    "sdk/servicebus/azure-messaging-servicebus-stress/**",
```

Comment:
Skip the cspell check for service bus stress test as it can't recognize several commands in Dockerfile and job.yaml.

Reply:
Sure, I'll do in this way!

Reply:
Changed